src/Nethermind/Nethermind.Evm/GasCostOf.cs

```diff
@@ -54,6 +54,8 @@ public static class GasCostOf
         public const long InitCodeWord = 2; //eip-3860 gas per word cost for init code size
 
         public const long ColdSLoad = 2100; // eip-2929
+        public const long ParentBeaconBlockPrecompile = ColdSLoad * 2; // Eip hardcded 4200 // eip-2929
```

Comment:
@Demuirgos Did you ask about clarification in EIP?

Reply:
yeah, he said that the gas to be used is mentioned in the spec and in the reasoning (I still feel conflicted abt it tho)

Reply:
Inconsistent naming sometimes it is `BeaconBlockRootPrecompile` sometimes `ParentBeaconBlockPrecompile`